platform-ui-skin/src/main/webapp/skin/less/_global.less

```diff
@@ -1162,6 +1162,11 @@ hr {
 .tertiary-color {
   color: @tertiaryColor!important;
 }
+
+.header-title-color {
+  color: @greyColorLighten1!important;
```

Comment:
we have already text-sub-title class that defines this property

Reply:
Done!

Reply:
Done!